amazon-s3/5.7/modules/ROOT/pages/amazon-s3-connector-config-topics.adoc

```diff
@@ -0,0 +1,166 @@
+
+[source,json,linenums]
+----
+{
+  "Version": "2012-10-17",
+  "Id": "arn:aws:sqs:REGION:ACCOUNT_ID:QUEUE_NAME/SQSDefaultPolicy",
+  "Statement": [
+    {
+      "Sid": "Sid1593770403887",
+      "Effect": "Allow",
+      "Principal": "*",
+      "Action": "SQS:SendMessage",
+      "Resource": "arn:aws:sqs:REGION:ACCOUNT_ID:QUEUE_NAME",
+      "Condition": {
+        "ArnLike": {
+          "aws:SourceArn": "arn:aws:s3:::BUCKET_NAME"
+        }
+      }
+    }
+  ]
+}
+----
+
+[[queue-created-by-source]]
+== Queue Is Created by Source
+
+If the queue is created by the source operation, the queue name has the prefix `MULE-S3-TRIGGER`.
+
+When the queue is created by the source, the user must have permission to both create a queue and set the queue attributes:
+
+[source,json,linenums]
+----
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": [
+                "sqs:DeleteMessage",
+                "sqs:ReceiveMessage",
+                "sqs:GetQueueAttributes",
+                "sqs:CreateQueue",
+                "sqs:SetQueueAttributes"
+                "sqs:GetQueueUrl"
+            ],
+            "Resource": "arn:aws:sqs:REGION:ACCOUNT_ID:MULE-S3-TRIGGER-*"
+        }
+    ]
+}
+----
+
+When the queue is created by the source operation, an SQS policy is added to the queue when the S3 object is created or deleted in the bucket in the following format:
+
+
+[source,json,linenums]
+----
+{
+  "Version": "2012-10-17",
+  "Id": "example-ID",
+  "Statement": [
+    {
+      "Sid": "example-statement-ID",
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": "*"
+    },
+      "Action": "SQS:SendMessage",
+      "Resource":QUEUE_ARN,
+      "Condition": {
+        "ArnLike": {
+          "aws:SourceArn": "BUCKET_ARN"
+        }
+      }
+    }
+  ]
+}
+----
+
+== S3 Policy Bucket
+
+For the S3 policy, use this format for both cases:
+
+[source,json,linenums]
+----
+{
+    "Version": "2012-10-17",
+    "Id": "Policy1593761427184",
+    "Statement": [
+        {
+            "Sid": "Stmt1593760119344",
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::ACCOUNT_ID:user/test"
+            },
+            "Action": "SPECIFIC_ACTION",
+            "Resource": "arn:aws:s3:::BUCKET_NAME/*"
+        },
+        {
+            "Sid": "Stmt1593760259223",
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::ACCOUNT_ID:user/test"
+            },
+            "Action": ["s3:PutBucketNotification", "s3:GetBucketNotification"],
+            "Resource": "arn:aws:s3:::BUCKET_NAME"
+        }
+    ]
+}
+----
+
+In this example, the *SPECIFIC_ACTION* is `s3:PutObject` or `s3:DeleteObject`, depending on the specified source operation.
```

Comment:
Suggestions to make it clear that SPECIFIC-ACTION is a variable: 

"Action": "<SPECIFIC_ACTION>"

I this example, <SPECIFIC_ACTION> is ...